src/main/java/org/mockito/internal/stubbing/answers/DefaultValidableAnswer.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2007 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+
+package org.mockito.internal.stubbing.answers;
+
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+
+import static org.mockito.internal.exceptions.Reporter.wrongTypeReturnedByDefaultAnswer;
+
+public class DefaultValidableAnswer {
```

Comment:
Actually this almost the same code as [`AnswerValidator.validateDefaultAnswerReturnedValue`](https://github.com/mockito/mockito/pull/826/files#diff-200d4fd7223f1f11f9971c8a3148ec42L99). I just renamed and inlined the code here. Look at the commit c588737f0e11dea3741abbcfd7205bd4eb77f913